odoo/addons/base/res/res_config.py

```diff
@@ -504,9 +507,11 @@ def default_get(self, fields):
                 res[name] = int(res[name])
 
         # other fields: call all methods that start with 'get_default_'
+        # and call the method 'get_default_fields'
         for method in dir(self):
```

Comment:
This `for` loop may be deprecated in favor of the new single method `get_default_fields`...
(You may log a warning when a getter method is actually called this way.)